app/components/ui/tale-tab-metadata/component.js

```diff
@@ -91,5 +198,47 @@ export default Component.extend({
       const tale = this.get('model').get('tale');
       tale.set('licenseSPDX', selected);
     },
+    
+    /* 
+    Adds a new, blank user to the component authors object, which
+    then gets added as a new row in the UI.
+    */
+    addNewAuthor() {
+      let newAuthor = {
+        'id': this.ID(),
+        'firstName': '',
+        'lastName': '',
+        'orcid':'',
+      }
+      this.taleAuthors.pushObject(newAuthor)
+    },
+
+    /* 
+    Removes an author from the component, and is consequentially removed
+    in the UI.
+    */
+    removeAuthor(id) {
+      let currentAuthors = this.get('taleAuthors');
+      let deletedAuthor = currentAuthors.find(o => o.id === id);
+      if(deletedAuthor){
+        let authorLocation = currentAuthors.indexOf(deletedAuthor);
+        if (authorLocation > -1) {
+          this.taleAuthors.removeAt(authorLocation);
+        }
+      }
+    },
+
+    /* 
+      Open the modal that the user sees when an error occurred or when validation failed.
+    */
+    openErrorModal() {
+      let selector = '.ui.metadata-error.modal';
+      $(selector).modal('setting', 'closable', true);
+      $(selector).modal('show');
+    },
```

Comment:
May be out-of-scope for this issue, as the entire Run > Metadata view is missing form validation, but I should note that in the future we may revisit this to add some more visual cues to the validation: https://semantic-ui.com/behaviors/form.html

This allows the user to more easily recognize and correct any fields with invalid values